<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Bottom Bar Plugins</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QToolButton" name="buttonUpPlugin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Up</string>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType" >
         <enum>Qt::UpArrow</enum>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="0" column="1" >
       <spacer name="verticalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <widget class="QToolButton" name="buttonAddPlugin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Add</string>
        </property>
        <property name="icon" >
         <iconset resource="resourceBar.qrc" >
          <normaloff>:/add.svg</normaloff>:/add.svg</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QToolButton" name="buttonDelPlugin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Remove</string>
        </property>
        <property name="icon" >
         <iconset resource="resourceBar.qrc" >
          <normaloff>:/remove.svg</normaloff>:/remove.svg</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="4" column="1" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>78</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0" >
       <widget class="QToolButton" name="buttonDownPlugin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Down</string>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType" >
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
      <item rowspan="4" row="1" column="0" >
       <widget class="QListWidget" name="pluginListView" >
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragEnabled" >
         <bool>true</bool>
        </property>
        <property name="dragDropMode" >
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="alternatingRowColors" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Bar Size:</string>
       </property>
       <property name="buddy" >
        <cstring>horizontalSlider</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestore" >
       <property name="text" >
        <string>Restore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonApply" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resourceBar.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>324</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel" >
     <x>281</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>191</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel" >
     <x>349</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setBarSize(int)</slot>
 </slots>
</ui>
